index.html

```diff
@@ -37,7 +37,7 @@ <h1 class="title">Version Control with Mercurial</h1>
 <p>This lesson shows how to use a popular open source version control system called Mercurial (also known as hg). It is widely used, both because it's easy to set up and because of a hosting site called <a href="http://bitbucket.org">Bitbucket</a>. No matter which version control system you use, the most important thing to learn is not the details of their more obscure commands, but the workflow that they encourage.</p>
 <blockquote>
 <h2>Prerequisites</h2>
-<p>FIXME A short paragraph describing what learners need to know before tackling this lesson.</p>
+<p>Students should so far understand how to interact with the UNIX shell - changing directory, reading/writing from/to a file, and using arguement(s) to modify commands - in order to begin learning version control.</p>
```

Comment:
I'll make this change later today. Thank you for letting me know how the HTML is generated, I think I knew that but didn't realize it until I read your message here.

On January 9, 2015, at 4:25 PM, Raniere Silva <notifications@github.com> wrote:

In index.html:

> @@ -37,7 +37,7 @@ <h1 class="title">Version Control with Mercurial</h1> > <p>This lesson shows how to use a popular open source version control system called Mercurial (also known as hg). It is widely used, both because it's easy to set up and because of a hosting site called <a href="http://bitbucket.org">Bitbucket</a>. No matter which version control system you use, the most important thing to learn is not the details of their more obscure commands, but the workflow that they encourage.</p> > <blockquote> > <h2>Prerequisites</h2> > -<p>FIXME A short paragraph describing what learners need to know before tackling this lesson.</p> > +<p>Students should so far understand how to interact with the UNIX shell - changing directory, reading/writing from/to a file, and using arguement(s) to modify commands - in order to begin learning version control.</p> 

Could you copy and paste your change into index.md? The HTML files are generated by Pandoc. After you change index.md you can

$ make preview $ git add -u $ git commit --amend $ git push -f origin gh-pages 

to update your pull request. If you need an explanation for any of the previous command don't be afraid to ask.

And what about use

Students should understand how to interact with the Unix Shell - e.g. changing the working directory - in order to begin this lesson. 

instead?

—
Reply to this email directly or view it on GitHub.￼